src/node/CoinsViewFactory.cpp

```diff
@@ -153,12 +157,13 @@ mmr::LeafSet::Ptr CoinsViewFactory::BuildAndValidateLeafSet(
 }
 
 mmr::MMR::Ptr CoinsViewFactory::BuildAndValidateOutputMMR(
+	const std::shared_ptr<libmw::IDBWrapper>& pDBWrapper,
```

Comment:
Make sure you're using spaces, not tabs please. When I get a chance, I'll set up clang-format so we can easily enforce this.